nemo/collections/asr/parts/mixins/mixins.py

```diff
@@ -895,3 +895,21 @@ def diarize(self, paths2audio_files: List[str], batch_size: int = 1) -> List[str
             Speaker labels
         """
         pass
+
+
+class IPLMixin:
```

Comment:
instead of this. Add another callbacks class with same functionality here: https://github.com/NVIDIA/NeMo/tree/main/nemo/utils/callbacks